velox/exec/Task.cpp

```diff
@@ -197,6 +198,28 @@ Task::~Task() {
   removeSpillDirectoryIfExists();
 }
 
+SplitsState& Task::getPlanNodeSplitsStateLocked(
+    const core::PlanNodeId& planNodeId) {
+  auto it = splitsStates_.find(planNodeId);
+  if (LIKELY(it != splitsStates_.end())) {
```

Comment:
nit: use FOLLY_LIKELY?

Reply:
@xiaoxmeng 
What's the difference?

Reply:
No, I found the some use cases use FOLLY some not. I feel it might be better to use FOLLY_LIKELY for all.